### ISSUE TITLE
Is it expected that PlexKodiConnect first sync take that long?

### ISSUE THREAD

**Issue by alscaldas** [croneter/PlexKodiConnect#1503]
## Help yourself
* I did try to restart Kodi :-)
Well... I restarted it once, when I tried the first initial sync and noticed it apparently was doing nothing.

* I checked the [PKC Frequently Asked Questions on the PKC wiki](https://github.com/croneter/PlexKodiConnect/wiki/faq)
Yes, I did.

* I did try to reset the Kodi database by going to `PKC Settings -> Advanced -> "Reset the database and optionally reset PlexKodiConnect"` and then hitting YES, NO
NO, because since I've just installed PKC in a new LibreELEC installation and I'm running it for the first time I didn't believe it was necessary.

* I did check the [existing issues on Github](https://github.com/croneter/PlexKodiConnect/issues)
Yes I did.

## Describe the bug
Not sure if it's a bug, in fact... I've read the wiki and I'm aware that the first sync is supposed to take longer than the following ones, but thing is it's been running for the last 24 hours and since it apparently produces no log messages nor has a progress indicator I have no idea whether it's working or not. 

Since I have no idea how is the PKC internal structure but I read it changes Kodi databases I tried to take a look to see if there were any changes either with their sizes or alteration times but there are none so far (which leaves me to believe that things aren't working as supposed or the changes are done just once the process is completed).

Just to give some further context I'm running LibreELEC 9.2.6 / Kodi Leia (instaled on a 32GB class 10 SD card) on a RockPro64 (RK3399) with 4GB RAM and trying to sync a - quite large, I believe - remote share with around 16000 movies and 4000 tv shows.

The following message is the only log I got on kodi.log since yesterday.

```
2021-05-25 11:31:56.958 T:2991051648  NOTICE: PLEX.default: plugin.video.plexkodiconnect started
2021-05-25 11:31:56.960 T:2991051648  NOTICE: PLEX.default: Requesting full library scan
```

And as shown bellow, it's almost 24 hours since the last log msg.
```
Nostromo:~ # LANG=C date
Wed May 26 10:10:00 -03 2021
```

Again, I apologize if this isn't the right place to post that. It's only that I didn't know where else I could ask...

## To Reproduce
1. Run KPC for the first time.
2. Run the initial sync.
3. There's neither a progress indicator nor log messages, so it's impossible to know if the process is working (which is still more confusing considering that's the first time I'm using PKC). 

## Expected behavior
Either some console log message or some progress that indicates that things are working. 

## You need to attach a KODI LOG FILE!
Well, there's no apparent bug, the tool seems to be working, I only 
    
I am aware that I can delete Plex tokens that I accidentially posted by following the [instructions on the PKC wiki](https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug#i-published-my-plex-token-to-some-forum-or-github-anyone-can-now-access-my-plex-server)

**Comment by croneter** [comment]
I need your entire DEBUG log, see https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug

Switch to debug logging, reboot Kodi and wait for a few minutes until you grab the log

**Comment by croneter** [comment]
Log please, @alscaldas - otherwise I can't help you

**Comment by alscaldas** [comment]
> I need your entire DEBUG log, see https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug
> 
> Switch to debug logging, reboot Kodi and wait for a few minutes until you grab the log

Hi, first off thank you so much for taking the time to look into my issue.

Sorry for the delay, thing is I decided to reinstall LibreELEC (as I was still using Kodi Leia I thought about using a completely clean install and upgrade it to Matrix). As soon as it's all up and running I'll produce the logs the way you instructed.

**Comment by alscaldas** [comment]
Hello. I'd like to report that apparently my issue has to do with a faulty sd card.

I installed the newest LibreELEC version available to my box from scratch (LibreELEC 9.95.4) and got a couple of weird errors. So I tried to recreate the boot image and install all over again but this time I couldn't even get a working installation.  

Which now that I think of would also explain some issues and warning log messages not related to PlexKodiConnect I'd been having on the old (newly installed Kodi Leia) install too.

I don't have another sd card to test so that untill I get a new one I'll have to abort it for now but given the circumstances I do believe it's not an PKC related issue.

Thank you for the prompt reply and the interest and sorry for opening a ticket before correctly evaluate the problem.
